header.php

```diff
@@ -12,10 +12,11 @@
 <html>
 
 <head>
-<link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Aboreto&family=Raleway:wght@100;400;700&display=swap" rel="stylesheet">
-<!-- Raleway font not used yet -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Aboreto&family=Raleway:wght@100;400;700&display=swap" rel="stylesheet">
+    <!-- Raleway font not used yet -->
+    <script src="https://kit.fontawesome.com/91efaef67a.js" crossorigin="anonymous"></script>
```

Comment:
We should include Fontawesome and Google Fonts from project, not from header links/scripts

Reply:
To exclude from header.php? 
And where to put this? To functions.php under load assets in header?